### PR TITLE
Add forms

### PR DESCRIPTION
**Pull request by ezraermy**
In this pull request, I:
- Created a method called current_user in your ApplicationController to make current user data available to all controllers.
- Created forms to perform the following functions:
  - Creates a Post on behalf of the current_user 
  - Create a comment on behalf of the current_user 
- Allow Users to add likes to Posts

### Other requirements 
- No linter errors
- Document professional way
- Follow gitflow

### CODE REVIEW

File: app/controllers/posts_controller.rb [hunk 1]
```diff
@@ -7,5 +7,30 @@ def index
...
+    else
+      flash[:error] = 'Post not created!'
+      render :new
+    end
+  end
+
+  def post_params
+    params.require(:post).permit(:title, :content)
   end
 end
```
Comment (Qoosim) [inline app/controllers/posts_controller.rb:38]: - [OPTIONAL] I would recommend you to make your post_params `private`. You can chek this [link](https://tjoye20.medium.com/ruby-access-control-basics-public-vs-private-vs-protected-methods-7788b26e04a7#:~:text=Private%20methods%20can%20only%20be,receiver%20is%20always%20implicitly%20self.) for more info.

File: app/controllers/comments_controller.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    else
+      flash[:alert] = 'Comment not created!'
+      render :new
+    end
+  end
+
+  def comment_params
+    params.require(:comment).permit(:text)
+  end
+end
```
Comment (Qoosim) [inline app/controllers/comments_controller.rb:24]: - [OPTIONAL] I would recommend you to make your comment_params `private`. You can chek this [link](https://tjoye20.medium.com/ruby-access-control-basics-public-vs-private-vs-protected-methods-7788b26e04a7#:~:text=Private%20methods%20can%20only%20be,receiver%20is%20always%20implicitly%20self.) for more info.

**Review by Qoosim — APPROVED** [review]
#  *Status: Approved* :tada: :heavy_check_mark: 

Hi @ezraermy, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
-  Implemented all the requirements. :heavy_check_mark: 
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct Gitflow. :heavy_check_mark: 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/controllers/comments_controller.rb (hunk 2 above)
Comment (ezraermy) [reply app/controllers/comments_controller.rb:24]: Thanks indeed for the feedback and tips.

**Pull request merged**